api/tests/unit/application/memberships/membership-controller_test.js

```diff
@@ -0,0 +1,178 @@
+const { expect, sinon, hFake, domainBuilder } = require('../../../test-helper');
+
+const membershipController = require('../../../../lib/application/memberships/membership-controller');
+const usecases = require('../../../../lib/domain/usecases');
+const membershipSerializer = require('../../../../lib/infrastructure/serializers/jsonapi/membership-serializer');
+const Membership = require('../../../../lib/domain/models/Membership');
+const requestResponseUtils = require('../../../../lib/infrastructure/utils/request-response-utils');
+
+describe('Unit | Controller | membership-controller', function () {
+  describe('#create', function () {
+    it('should call createMembership usecase', async function () {
```

Comment:
**suggestion**
On pourrait dire ici `should return the serialized created membership` et changer l'expect avec : 
```
const serializedMembership = Symbol('membership serialized')
membershipSerializer.serialize.withArgs(membership).returns(serializedMembership)

// when
const result = await membershipController.create(request, hFake);

// then
expect(result.source).equal(serializedMembership);
 ```
 
 Comme ça on teste aussi que le retour de la réponse est serializé.

Reply:
Un test a ajouter plutôt non ? on garde les tests qui vérifient que les usecases sont bien appelés avec les bons paramètres

Reply:
vu ensemble => un test pour verifier le retour du serialize mais qui vérifie aussi les paramètres du usecase avec un withArgs dans le stub.
Ajout d'un expect pour vérifier qu'il est appelé une fois.